README.rst

```diff
@@ -43,3 +43,37 @@ https://rockcraft.readthedocs.io
     :scale: 100%
     :target: https://snapcraft.io/rockcraft
   
+Testing
+-------
+
+In addition to unit tests in :code:`tests/unit`, which can be run with :code:`make test-units`,
+a number of integrated tests in :code:`tests/spread` can be run with `Spread`_. See the
+`general notes`_ and take note of these ``rockcraft``-specific instructions:
+
+* Initialize/update git submodules to fetch Spread-related helper scripts:
+
+  .. code-block::
+
+     $ git submodule init
+     $ git submodule update
+
+* Spread needs a ``rockcraft`` snap in order to run. Create one with :code:`snapcraft` via:
+
+  .. code-block::
+
+     $ snapcraft --use-lxd
+     $ cp <generated snap> tests/
+
+* Run Spread (the :code:`google` backend needs access to :code:`snapd-spread`):
```

Comment:
I am not sure about documenting snapd-spread as we don't mention how to get access to it. The multipass or qemu providers do not require this.